README.md

```diff
@@ -171,3 +434,19 @@ $ cat > templates/xml/my_mentoring_scenario.xml
 
 Restart the workbench to take the new scenarios into account.
 
+Upgrading from Version 1
+------------------------
+
+To upgrade a course from the earlier version of this XBlock, run the following
+command on a system with edx-platform and xblock-mentoring installed:
+
+```bash
+$ SERVICE_VARIANT=cms DJANGO_SETTINGS_MODULE="cms.envs.devstack" python -m mentoring.v1.upgrade "Org/Course/Run"
```

Comment:
@antoviaque Fixed setup.py so the upgrade script should work now however it's installed.